### PR TITLE
chore: Update Vega version

### PR DESCRIPTION
**Pull request by TomMcL**
### Description

Updating Vega core version + minor associated tweaks

### Testing

Tests pass

### Breaking Changes

Should be none

### CODE REVIEW

File: scripts/update_vega_version.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ VEGA_SIM_NETWORKS_TAG=master" > .env
 
 make
 make proto
-pytest -s -v --log-cli-level INFO -m integration
+# pytest -s -v --log-cli-level INFO -m integration
```
Comment (cdummett) [inline scripts/update_vega_version.sh]: Is this meant to be commented out still?

Comment (TomMcL) [reply scripts/update_vega_version.sh]: It is not

Comment (TomMcL) [reply scripts/update_vega_version.sh]: :+1:

**Review by cdummett — APPROVED** [review]
LGTM

**Pull request merged**